tests/client/core/reducers/test_api.js

```diff
@@ -11,15 +11,16 @@ describe('api reducer', () => {
     assert.strictEqual(api(state, { type: 'UNRELATED' }), state);
   });
 
-  it('stores the JWT', () => {
-    const token = 'some.base64.STRING';
-    assert.deepEqual(api({ foo: 'bar' },
-      { type: 'SET_JWT', payload: { token } }), { foo: 'bar', token });
+  it('stores the auth token', () => {
+    const token = userAuthToken();
+    assert.deepEqual(
+      api({ foo: 'bar' }, actions.setAuthToken(token)),
+      { foo: 'bar', token });
   });
 
-  it('clears the JWT on log out', () => {
+  it('clears the auth token on log out', () => {
     assert.deepEqual(
-      api({ lang: 'fr', clientApp: 'firefox', token: 'secret' },
+      api({ lang: 'fr', clientApp: 'firefox', token: 'user:auth:token' },
```

Comment:
Dunno if using `signedInApiState` here makes more sense, but I guess it doesn't buy us anything. Just consistency, really.